backend/stakepoold/config.go

```diff
@@ -264,6 +272,10 @@ func loadConfig() (*config, []string, error) {
 		PoolFees:   defaultPoolFees,
 		RPCKey:     defaultRPCKeyFile,
 		RPCCert:    defaultRPCCertFile,
+
+		Prometheus:       defaultPrometheus,
```

Comment:
I don't think it's necessary to specify false since that is the zero value of a bool, but it's also fine if you want to specify it.  Just pointing that out, not asking for a change.